meson.build

```diff
@@ -226,10 +226,6 @@ libgcab = dependency('libgcab-1.0', version : '>= 1.0', fallback : ['gcab', 'gca
 gcab = find_program('gcab', required : true)
 bashcomp = dependency('bash-completion', required: false)
 python3 = find_program('python3')
-tpm2tss = dependency('tss2-esys', version : '>= 2.0', required: false)
-if tpm2tss.found()
```

Comment:
Don't you still need this config.h define?

Reply:
No, take a look at rest of meson.build to see how I did it with the empty `dependency('', required:false)`

Reply:
I'm not sure how that defines `ifndef HAVE_TSS2`

Reply:
Oh I see what you mean and my mistake.  Pushed a fix.